kolibri/core/assets/src/vue/side-nav/learn.svg

```diff
@@ -0,0 +1,4 @@
+<svg fill="#996189" height="24" viewbox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Added a "technical debt" card to track this for later:
https://trello.com/b/bjHannKQ/kolibri-tasks